ArrtSource/Storage/UI/StorageBrowserModel.cpp

```diff
@@ -281,6 +281,17 @@ bool StorageBrowserModel::IsSrcAsset(const QString& file)
     return formats.indexOf(extension) >= 0;
 }
 
+bool StorageBrowserModel::IsPointCloud(const QString& file)
+{
+    QString formats = ";e57;ply;xyz;las;laz;";
+    formats = formats.toLower();
+
+    QFileInfo info(file);
+    QString extension = ";" + info.suffix().toLower() + ";";
+
+    return formats.indexOf(extension) >= 0;
```

Comment:
Here is a different way of doing the extension test (maybe the more canonical way):

QStringList singleFileExtensions;     
singleFileExtensions << "e57" << "ply" << "xyz" ... ;
singleFileExtensions.contains(info.suffix().toLower());

(involves more allocations than your code, but somehow looks 'cleaner' ;-) )

Reply:
Keeping the original method for consistency with IsSrcAsset